js/offer.js

```diff
@@ -0,0 +1,80 @@
+import {getHouseRentalCount} from './random-object.js';
+const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
+const templateFRAGMENT = document.createDocumentFragment();
+const announcementMap = document.querySelector('.map__canvas');
+
+const mapTypeHousing = {
+  flat: 'Квартира',
+  bungalow: 'Бунгало',
+  house: 'Дом',
+  palace: 'Дворец',
+  hotel: 'Отель'
+};
+
+
+const getRandomFeatures = (array, list) => {
```

Comment:
what is array? what is list? i think that array is a list. Rename params, i should understand your code, now i don't. Also rename function, you named this function as `getRandomFeatures`, but it returns nothing